scripts/submit_skims.sh

```diff
@@ -303,7 +307,7 @@ SemiLepXSec=`echo "791.0 * 2 * (1-0.6741) * 0.6741" | bc`
 DATA_MAP=(
     ["TTToHadronic"]="-n 20 -x ${FullyHadXSec}"
     ["TTTo2L2Nu"]="-n 100 -x ${FullyLepXSec}"
-    # ["TTToSemiLeptonic"]="-n 100 -x ${SemiLepXSec}"
+    ["TTToSemiLeptonic"]="-n 100 -x ${SemiLepXSec}"
```

Comment:
I think the following samples are missing in the list below: WWW,WWZ,WZZ,ZZZ, TTWH, TTZH

Reply:
Good point. It seems we never created the 3 VVV big ntuples, but this is a nice moment to do so. As for TTWH and TTZH, I do not see them in [our list of samples](https://github.com/bfonta/LLRHiggsTauTau/blob/aa13e6859497a7da2a856c26bdb2fb60aff41e37/NtupleProducer/test/datasets_UL18.txt) (taking 2018 as an example), but [they were used](https://github.com/bfonta/LLRHiggsTauTau/blob/aa13e6859497a7da2a856c26bdb2fb60aff41e37/NtupleProducer/test/datasets_Legacy2018.txt#L108-L109) in the non-resonant analysis. 
I would also need all of these processes' cross-sections. Do you by any chance have those at hand?

@portalesHEP FYI

Reply:
good catch indeed, TTWH & TTZH can be found [here](https://cmsweb.cern.ch/das/request?view=list&limit=50&instance=prod%2Fglobal&input=dataset%3D%2FTT*H_TuneCP5_13TeV-madgraph-pythia8%2FRunIISummer20UL18MiniAODv2*%2FMINIAODSIM) for UL18. We can use the XSDB cross-sections for those: 0.001143 pb ([TTWH](https://xsdb-temp.app.cern.ch/?searchQuery=DAS=TTWH_TuneCP5_13TeV-madgraph-pythia8)) and 0.001136 pb ([TTZH](https://xsdb-temp.app.cern.ch/?searchQuery=DAS=TTZH_TuneCP5_13TeV-madgraph-pythia8))

Reply:
Yes I do :)

TTWW 0.006979
TTWZ 0.002441
TTWH 0.00158
TTZH 0.00154
TTZZ 0.001386

ZZZ 0.0147
WZZ 0.057 
WWW 0.209 
WWZ 0.168

Reply:
Well apparently some of them changed after we finished the non-resonant analysis eheh

Reply:
We're probably fine with either value though; the cross-section is tiny, and the contribution to our selection even tinier considering those are inclusive samples. Anyway since we have them now, we can use the XSDB values for TTVH.

Reply:
I've just added the missing samples, here and in the [big ntuples' framework](https://github.com/LLRCMS/LLRHiggsTauTau/pull/160).